src/pool/srv_pool.c

```diff
@@ -2386,7 +2386,9 @@ ds_pool_update_internal(uuid_t pool_uuid, struct pool_target_id_list *tgts,
 	}
 
 	updated = true;
-	replace_failed_replicas(svc, map);
+
+	/** TODO: Call disabled; enable along with `rsvc_client` changes **/
+	(void) replace_failed_replicas;
```

Comment:
Is this just to avoid compiler warning of unused static function?

Reply:
Yes.